handlers/config_update.go

```diff
@@ -39,24 +38,19 @@ func (h *ConfigUpdateHandler) ConfigUpdate(event *revents.Event, cli *client.Ran
 	env = append(env, fmt.Sprintf("%v=%v", "CATTLE_HOME", home))
 	args := itemNames
 
-	retcode := -1
+	retcode := 0
 
 	command := exec.Command(config.Sh(), args...)
 	command.Env = env
 	command.Dir = home
-	output, err := command.Output()
+	output, err := command.CombinedOutput()
 	if err != nil {
-		logrus.Error(err)
-	} else {
-		retcode = 0
+		retcode = 1
 	}
-	if retcode == 0 {
-		return reply(map[string]interface{}{
-			"exitCode": retcode,
-			"output":   string(output),
-		}, event, cli)
+	if retcode != 0 {
+		pro := &progress.Progress{Request: event, Client: cli}
+		pro.Update("config update failed", string(output))
```

Comment:
In the situation of an error the response needs to be `Transitioning: "error"` and the `exitCode` should be set in the response too.